docs/src/pages/PhoneInputPage/PhoneInputPage.js

```diff
@@ -80,10 +80,10 @@ class StoryContainer extends Component<Props, State> {
         value={this.state.value}
         dialingCode={this.state.dialingCode}
         dialingCodes={[
-          { code: '1', description: '+1 (US)' },
-          { code: '2', description: '+1 (CA)' },
-          { code: '44', description: '+44' },
-          { code: '55', description: '+55' },
+          { code: '1', description: '+1 (US)', numberPrefix: '+1' },
```

Comment:
Do the codes need to be aligned? E.g. `1_us`, `1_ca`, `44_uk`?

Reply:
As long as each is unique it doesn't matter, but can't do any harm 👍

Reply:
👍